src/constants_and_utils/utils.js

```diff
@@ -17,8 +19,46 @@ function waitForElementToBeClickable(htmlElement, timeoutInMilliseconds, errorMe
   browser.wait(EC.elementToBeClickable(htmlElement), timeoutInMilliseconds, errorMessage);
 }
 
+function protractorHelperWarning() {
+  return colors.yellow("Protractor-helper warning: ");
+}
+
+function readMoreDoc() {
+  return colors.cyan(" [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]");
+}
+
+function obsoleteFunction(functionName, text = "") {
+  console.warn(
```

Comment:
Maybe we could use ES6 syntax for string concatenation here too.

Reply:
I will do it.